src/test/java/com/bulkbuy/enterprise/EnterpriseApplicationTests.java

```diff
@@ -44,7 +39,7 @@ void verifyOrderProperties() {
         int orderId = 52;
         int quantity = 5;
         String productDescription = "Plain Paper";
-        Date orderDate = new Date(2021,10,1);
+        Date orderDate = new Date(2021, Calendar.NOVEMBER,1);
```

Comment:
when possible replace hard coded values with constants defined within java
https://www.baeldung.com/java-clean-code

Reply:
Good one.  Especially with months, since they are 0 based in Java (!?!?!?!), they tend to be error prone.